.github/workflows/merge-me.yml

```diff
@@ -0,0 +1,29 @@
+name: 'dhis2: auto-merge (dependabot)'
+
+on:
+    check_suite:
+        types:
+            - completed
+    pull_request:
+        types:
+            - edited
+            - labeled
+            - opened
+            - ready_for_review
+            - reopened
+            - synchronize
+            - unlabeled
+            - unlocked
+
+jobs:
+    merge-me:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Merge me!
+              uses: ridedott/merge-me-action@master
+              with:
+                  # This must be used as GitHub Actions token does not support
+                  # pushing to protected branches.
+                  GITHUB_LOGIN: dependabot
```

Comment:
Is `dependabot` a valid user here?  Should it not be `dhis2-bot`?

Reply:
The idea is to allow `dependabot` to authenticate with the token for `dhis2-bot` if I understand the workaround correctly. `dependabot` is the default value if omitted, but I figure that it is preferable to be explicit.

The `GITHUB_LOGIN` is used to determine which PRs from a user to merge, so if we want to automatically merge PRs by `dhis2-bot` we should put `dhis2-bot` as `GITHUB_LOGIN`. In this case we want to merge PRs by `dependabot`.

We may want to use a another action for auto-merging e.g. translations and/or other PRs from dhis2-bot in the future.

Reply:
Ah got it, that makes sense!